src/gui/windows/settings.zig

```diff
@@ -22,6 +22,7 @@ pub fn onOpen() void {
 	list.add(Button.initText(.{0, 0}, 128, "Controls", gui.openWindowCallback("controls")));
 	list.add(Button.initText(.{0, 0}, 128, "Advanced Controls", gui.openWindowCallback("advanced_controls")));
 	list.add(Button.initText(.{0, 0}, 128, "Change Name", gui.openWindowCallback("change_name")));
+	list.add(Button.initText(.{0, 0}, 128, "Controller mappings", gui.openWindowCallback("controller_mappings_settings")));
```

Comment:
I think this is cluttering the settings menu. How about putting this button inside the controls window, when the gamepad controls are edited.